chirp/drivers/ft450d.py

```diff
@@ -29,9 +30,16 @@
 
 LOG = logging.getLogger(__name__)
 
-CMD_ACK = 0x06
-# TBD: Enable some form of generated UI field, for the memory tags
-# That field wiould not be stored in img file, but generated in get_memory
+HAS_FUTURE = True
+try:                         # PY3 compliance
+    from builtins import bytes
+    from past.builtins import range     # RJD 12/18/22
+except ImportError:
+    HAS_FUTURE = False
+    LOG.warning('python-future package is not '
+                'available; %s requires it' % __name__)
+

```

Comment:
Why are you adding this to the py3 branch? The behavior of `bytes` in python3 is natively what it is with the future package on python2. Using future to allow a driver to work on both is good, but at this point, these changes are all python3 so adding this code doesn't do anything useful.